### PR TITLE
Debug all the way!

### PR DESCRIPTION
**Pull request by ChielBruin**
- Added more nodes:
  - `battery_display`: a node that shows the battery charge
  - `ultrasonic_display`: a node that shows the distances measured by the ultrasonic sensors
- Improved `keyboard_sensors`
- Improved package structure
- Added 2 launch profiles:
  - `display.launch`: launches all display nodes
  - `turtle.launch`: launches the turtle and the turtleBridge

### CODE REVIEW

**Comment by marcelkuipers** [comment]
maybe it is nicer to reduce battery charge with a discrete amount, say 5 or 10 percent each time. Right now you have to press a lot of times before the charge gets low. Also, the charge will never become zero, do we want this?

**Comment by marcelkuipers** [comment]
for the rest it looks good

**Comment by ChielBruin** [comment]
Good point! I changed it

File: catkin_ws/src/acr_debug/src/keyboard_sensors/keyboard_sensors.py [hunk 1]
```diff
@@ -95,80 +124,60 @@ def getKey():
 	rospy.init_node('acr_keyboard')
 
 	try:
-		print msg
-		poi = float('nan')
-
-		ult_max = 10000
-		ult_dist = 1999
+		ult_max = 500
+		ult_dist = 100
```
Comment (marcelkuipers) [inline catkin_ws/src/acr_debug/src/keyboard_sensors/keyboard_sensors.py:128]: You should also update the min distance attribute in sensordata.hpp. This attribute is set to 2000, causing the SensorData class to always think there is something right in front of the robot.

**Comment by marcelkuipers** [comment]
okay, it looks good now

**Pull request merged**